README.md

```diff
@@ -35,6 +35,16 @@ var app = new EmberApp({
 });
 ```
 
+You can also opt out of including the fetch polyfill, if you do not need to run your tests in older browsers:
```

Comment:
Should this be "if you do not need to test `fetch` with pretender"? Since the polyfill is not for older browsers but to make `fetch` interceptable by pretender.